ACadSharp.Tests/ACadSharp.Tests.csproj

```diff
@@ -12,6 +12,7 @@
     <PackageReference Include="MSTest.TestAdapter" Version="2.1.1" />
     <PackageReference Include="MSTest.TestFramework" Version="2.1.1" />
     <PackageReference Include="coverlet.collector" Version="1.3.0" />
+    <PackageReference Include="Newtonsoft.Json" Version="9.0.1" />
```

Comment:
This package is not needed if we are using `netcoreapp3.1`

Reply:
Once again, this is was an items copied from #94 by accident.  Will fix.